be/src/exprs/vectorized/math_functions.h

```diff
@@ -161,10 +164,22 @@ class MathFunctions {
     DEFINE_VECTORIZED_FN(round_up_to);
 
     /**
-     * @param: [DoubleColumn, IntColumn]
-     * @return: DoubleColumn
+     * @param: [DecimalV3Column<int32_t>, IntColumn]
+     * @return: DecimalV3Column<int128_t>
+     */
+    DEFINE_VECTORIZED_FN(truncate_decimal32);
```

Comment:
What if other double functions? All of them need to implement three variants?

Reply:
Currently, yes, we need to add three variants